### PR TITLE
Fix condition_visits PK naming

### PR DESCRIPTION
**Pull request by t-young31**
## Reason for changes and summary

The primary key in ConditionVisits doesn't conform to the naming convention in star. Apologies for introducing this in the first place – glad it's not in main yet!

### CODE REVIEW

File: inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/conditions/ConditionVisits.java [hunk 1]
```diff
@@ -28,17 +28,17 @@
 @NoArgsConstructor
 @ToString(callSuper = true)
 @Table(indexes = {@Index(name = "vo_hospital_visit_id", columnList = "hospitalVisitId"),
-        @Index(name = "vo_condition_visit_id", columnList = "conditionVisitId")})
+        @Index(name = "vo_condition_visit_id", columnList = "conditionVisitsId")})
 public class ConditionVisits {
 
     /**
      * \brief Unique identifier in EMAP for this link record.
```
Comment (stefpiatek) [inline inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/conditions/ConditionVisits.java:35]: Probably should reword this as well maybe?

**Review by stefpiatek — APPROVED** [review]
Good spot, interesting though, I thought we had an automated test which should pick this up - do we know why that didn't (or was that only for foreign keys)?

**Comment by t-young31** [comment]
> or was that only for foreign keys?

think it was just FKs. Worth adding an issue?
